### DISCUSSION TITLE
Encryption

### DISCUSSION THREAD

**Discussion by ghost** [isar/isar#1366] [Help] [answered]
Hello, I have a few questions regarding encryption:

- will encryption also be supported for web?
- will encryption only work with sqlite or also with the native Isar engine?

**Comment by simc** [comment, marked as answer]
It may be supported on Web in the future. Not sure yet.

Only the sqlite engine will support encryption

**Reply by itsJoKr** [reply]
Wondering why Isar engine can't support the encryption too?

I'm asking because inserting 300k items takes
- 23 seconds in IsarEngine.sqlite (same with or without encryption, but seems like indexes are affecting the speed)
- 1.5 seconds in IsarEngine.isar

Reads are comparable in speed.
